### PR TITLE
#165273478: rate article out of 5

### PR DESCRIPTION
**Pull request by 3Nakajugo**
#### What does the PR do?
- enables the user to rate an article out of 5

#### Tasks to be completed. 

- create app rate_article
- make migrations
- update authors/URLs.py
- add rate_article to installed apps
- create tests directory
- add serializers.py
- add base tests
- add fixtures
- lint code

#### how to test it 
- relevant endpoint
 `https://inviticus-staging-pr-22.herokuapp.com/api/articles/<article_id>/rating/`

- example of body
```
{
	"rating":4
}
```
- response body
```
{
    "rating": {
        "rating": 2,
        "user": "ednav"
    }
}
```

#### relevant stories
[#165273478](https://www.pivotaltracker.com/story/show/165273478)

### CODE REVIEW

File: authors/apps/rate_article/serializer.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    def validate_rating(self, rating):
+        if rating < 1 or rating > 5:
+            raise serializers.ValidationError("rating should include numbrs 1 to 5")
+        return rating
+
+    class Meta:
+
+        model = Rate
+
+        fields = ('rating', 'user')
```
Comment (joelmugaya) [inline authors/apps/rate_article/serializer.py]: You need to add a line at the bottom of this file.
About the blank lines between this code, there is code that needs compacting. Not only in this file.

**Review by mwinel — APPROVED** [review]
This is ready for a merge.

**Review by sanya-kenneth — APPROVED** [review]
Good work Edna

**Review by archibishop — APPROVED** [review]
@3Nakajugo LGTM I am merging this.

**Pull request merged**